src/c3chart/Area.js

```diff
@@ -8,11 +8,11 @@
 }(this, function (CommonND) {
     function Area(target) {
         CommonND.call(this);
-        this._class = "c3chart_Area";
 
         this._type = "area";
     };
     Area.prototype = Object.create(CommonND.prototype);
+    Area.prototype._class += " c3chart_Area";
```

Comment:
Yes - The class for the Area chart will now be ".common_Widget .c3chart_Common .c2chart_CommonND .c3chart_Area" - in the few instances we specifically needed this previously, the class was appened manually in the "enter" method.